webpack/components/BreadcrumbRouting/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const withRouter = (WrappedComponent) => {
```

Comment:
@sharvit - Instead of using inheritance I've used HoC here as we discussed about it earlier.
According to react documentation - this is the right pattern for extending component logic.